.github/workflows/prototyping.yml

```diff
@@ -8,12 +8,18 @@ on:
     paths:
       - '.github/workflows/**'
       - 'prototyping/**'
-      - 'Analysis/src/JsonEncoder.cpp'
+      - 'Analysis/**'
```

Comment:
It would be cleaner to move JsonEncoder to Ast, but this is good enough for now probably.